CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+There are incompatible ABI changes in this release. The clli member was added
+to the avifImage struct.
+
 ### Added
 * Add STATIC library target avif_internal to allow tests to access functions
   from internal.h when BUILD_SHARED_LIBS is ON.
+* Add clli metadata read and write support
```

Comment:
https://github.com/AOMediaCodec/libavif/pull/1194 says "Add clli metadata read support", but I think it actually added write support also, right?